spring-boot-autoconfigure/pom.xml

```diff
@@ -497,5 +497,11 @@
 			<artifactId>slf4j-jdk14</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.cxf</groupId>
+			<artifactId>cxf-bundle</artifactId>
+			<version>3.0.0-milestone2</version>
```

Comment:
Sounds like we should go with 3.0.0, but we'll have to wait until 3.0.0 has GAed before merging.